browser/templates/browser/person.html

```diff
@@ -1,9 +1,38 @@
 {% extends "browser/base.html" %}
 {% load app_filters %}
-
 {% block title %}{{ person.name }} (Person){% endblock %}
 
+
 {% block main %}
+
+<script>
+        $(document).ready(function() {
+            var research_id = "";
+
+            $('[data-curation-tooltip]').tooltip();
+
+            $(document).on("click", "#open-InvestigatorDeleteModal", function () {
+                var id = $(this).data('id');
+                var subject = $(this).data('subject');
+                var role = $(this).data('role');
+                var year = $(this).data('year');
+                research_id= $(this).data('id');
+
+                $(".modal-body #investigator_delete_modal_year").text(year);
+                $(".modal-body #investigator_delete_modal_role").text(role);
+                $(".modal-body #investigator_delete_modal_subject").text(subject);
+
+            });
+
+            $(document).on("click", "#investigator_delete_modal_deleteButton", function () {
+                window.location = 'investigator/'+ research_id+'/delete/' ;
```

Comment:
the researcher id should be coming from the clicked button (not as a global variable)

Reply:
and it needs to be a post request

Reply:
I have made the delete investigator as a post request. However, I have retrieved the research_id field in a hidden input text field. Is this an acceptable approach ?

Reply:
Depends on how you actually implemented it. Just make a new PR when you're done and I'll look at it.